### PR TITLE
http-prompt: update to version 2.1.0

### PR DESCRIPTION
**Pull request by tsabirgaliev**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 11.2.3 20D91
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: net/http-prompt/Portfile [hunk 1]
```diff
@@ -15,32 +15,33 @@ long_description    ${description}
...
 
 python.link_binaries_suffix
 
 depends_test-append port:py${python.version}-mock \
                     port:py${python.version}-pexpect \
                     port:py${python.version}-pytest \
-                    port:py${python.version}-pytest-cov
+                    port:py${python.version}-pytest-cov \
+                    port:py${python.version}-wheel \
+                    port:py${python.version}-twine
```
Comment (lbschenkel) [inline net/http-prompt/Portfile:43]: Please keep this sorted alphabetically.

**Review by lbschenkel — CHANGES_REQUESTED** [review]
Thank you for the submission. Can you please split this into 2 commits, one for each port?

**Comment by tsabirgaliev** [comment]
> Thank you for the submission. Can you please split this into 2 commits, one for each port?

done

**Review by lbschenkel — APPROVED** [review]

**Comment by lbschenkel** [comment]
I'm approving it, but if you want you can include the other commit in this PR. I didn't mean that it should be 2 different PRs, only 2 different commits. However, if you want me to merge as-is and submit a different PR, then I'll go ahead and merge.

**Comment by tsabirgaliev** [comment]
@lbschenkel the two changes are actually unrelated, I can create a separate PR

**Comment by lbschenkel** [comment]
OK. Merging. Thanks again for your contribution.

**Pull request merged**